.gitignore

```diff
@@ -3,3 +3,8 @@
 *.sublime-project
 *.sublime-workspace
 *~
+bazel-bin
```

Comment:
these should probably have a leading `/`.
also, any reason not to just ignore `/bazel-*`?